blockchain/contracts/FlatNFTtheWorld.sol

```diff
@@ -1399,6 +1405,29 @@ contract NFTtheWorld {
         }
     }
 
+
+    // to be called automatically from backend
+    // checks weither reservation has timed out & if so, if reservedFor != owner, meaning it wasnt bought
+    // if so, reinstate as if Drop was executed but NFT wasnt reserved 
+    function revertTimedoutReservations(uint256 _dropHash) public returns (uint256){
```

Comment:
Maybe we should make this function executable only by admins so that the admins need to pay the fee and not some random person?

Reply:
Yes your right, but I wouldn't do so.
One Senario: 
A company plans a drop in the future, but meanwhile goes bankrupt.
Then you are able to cheat and reserve many NTFs (with different wallets).
After the drop you only pay one requested batch, all other bought NFTs got lost / locked.
The few NFTs you've got would have gained rarity and value.